src/app/features/landing-page/landing-page.component.ts

```diff
@@ -8,4 +10,8 @@ import { RouterLink } from '@angular/router';
   styleUrls: ['./landing-page.component.scss'],
   imports: [RouterLink],
 })
-export class LandingPageComponent {}
+export class LandingPageComponent {
+  constructor(title: Title) {
+    title.setTitle(convertTitle());
+  }
```

Comment:
you mix constructor and `inject` approaches. I remember that you mentioned "when I have a constructor, no need" but having the same approach across an application makes it more simple to read code. So maybe worth to change